subprojects/dependency-management/src/main/java/org/gradle/internal/component/local/model/DefaultLocalComponentMetadata.java

```diff
@@ -253,6 +248,11 @@ public AttributesSchemaInternal getAttributesSchema() {
         return attributesSchema;
     }
 
+    @Override
```

Comment:
So the local component itself has no attributes, but the variants of this component may have attributes. I'd appreciate a javadoc comment.